tests/lax_numpy_test.py

```diff
@@ -1752,6 +1752,25 @@ def testTrimZeros(self, a_shape, dtype, trim):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=True)
 
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape={}_rank{}".format(
+        jtu.format_shape_dtype_string(a_shape, dtype), rank),
+       "dtype": dtype, "a_shape": a_shape, "rank": rank}
+      for rank in (1, 2)
+      for dtype in default_dtypes
+      for a_shape in one_dim_array_shapes))
+  def testPoly(self, a_shape, dtype, rank):
+    if dtype in (np.float16, jnp.bfloat16, np.int16):
+      self.skipTest(f"{dtype} gets promoted to {np.float16}, which is not supported.")
+    elif rank == 2 and jtu.device_under_test() in ('tpu', 'gpu'):
+      self.skipTest("Nonsymmetric eigendecomposition is only implemented on the CPU backend.")
+    rng = jtu.rand_default(self.rng())
+    tol = { np.int8: 1e-4, np.int32: 1e-3, np.float32: 1e-3, np.float64: 1e-6 }
```

Comment:
A couple more tolernace updates are needed for the larger test suite:
- change the int8 tolerance to 1e-3
- on TPU, it looks like int32 & float32 tolerances need to be `0.1` for the tests to pass

Reply:
I've updated the tolerances in https://github.com/google/jax/commit/e985c135764c65f831235d9a4a6f16bc368670bb. I made the int32 and float32 tolerances specific to the TPU in [this line](https://github.com/google/jax/commit/e985c135764c65f831235d9a4a6f16bc368670bb#diff-6413a73e50360a19ffcf338449b206c057e47f6d66eb0a40be5610e6baff6420R1769).